packages/web/stores/root.ts

```diff
@@ -85,6 +85,10 @@ export class RootStore {
 
   public readonly profileStore: ProfileStore;
 
+  // NOTIFI QUESTION - We didn't want to modify the store here, but didn't see a better place to have
```

Comment:
@jonator not ideal, but this was the only way we could come up with sharing bidirectional state with Notifi context, and any possible transactions that are being executed